pytim/observables/profile.py

```diff
@@ -167,7 +167,7 @@ def _determine_bins(self):
         # we need to make sure that the number of bins is odd, so that the
         # central one encompasses zero (to make the delta-function
         # contribution appear always in this bin)
-        if (nbins % 2 > 0):
+        if (nbins % 2 == 0):
```

Comment:
Did you check explicitly that you get the delta contribution in the central bin like this? I understand the old code seems wrong if one reads the comment above it, but maybe there's some catch somewhere. There should be some simple configuration/test to check this. If not, we could add one.